stdlib/Profile/src/Profile.jl

```diff
@@ -1338,7 +1342,7 @@ function take_heap_snapshot(all_one::Bool=false; dir::Union{Nothing,S}=nothing)
     else
         fpath = joinpath(expanduser(dir), fname)
     end
-    return take_heap_snapshot(fpath, all_one)
+    return take_heap_snapshot(fpath, all_one; redact_data=redact_data, streaming=streaming)
```

Comment:
and then pass them through, here:
```suggestion
    return take_heap_snapshot(fpath, all_one; kwargs...)
```